lms/services/digest.py

```diff
@@ -87,6 +87,25 @@ def send_instructor_email_digests(  # pylint:disable=too-many-arguments
             )
 
 
+@dataclass(frozen=True)
+class Annotation:
+    """Info about an annotation from the h API."""
+
+    userid: str
+    authority_provided_id: str
+
+    @classmethod
+    def make(cls, annotation_dict):
+        """Make an Annotation from an annotation dict from the h bulk annotation API."""
+        userid = annotation_dict["author"]["userid"]
+        authority_provided_id = annotation_dict["group"]["authority_provided_id"]
+
+        return cls(
+            userid=userid,
+            authority_provided_id=authority_provided_id,
+        )
```

Comment:
The code formatting is more verbose than it needs to be here, but it's going to make more sense when we add more fields and defensive coding later, see https://github.com/hypothesis/lms/pull/5755